etc/pdfsam.profile

```diff
@@ -24,6 +24,7 @@ protocol unix
 seccomp
 shell none
 
+private-bin pdfsam,sh,bash,archlinux-java,grep,awk,dirname,uname,which,sort,find,readlink,expr,ls,java-config
```

Comment:
What is archlinux-java?
I would expect that for all other distributions `java` is required for it to work. Can you please add that?
Has archlinux really no plain java command?